2020/01/10 15:28:34 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "localhost"
2020/01/10 15:28:34 [warn] 6#6: *1 upstream server temporarily disabled while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "localhost"
2020/01/10 15:28:34 [crit] 6#6: *1 connect() to [::1]:9000 failed (99: Address not available) while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://[::1]:9000", host: "localhost"
2020/01/10 15:28:34 [warn] 6#6: *1 upstream server temporarily disabled while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://[::1]:9000", host: "localhost"
2020/01/10 15:28:35 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:28:35 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:28:35 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:28:36 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:28:36 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:28:36 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:28:36 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:28:36 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:28:36 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:28:37 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:28:37 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:28:37 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:28:38 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:28:38 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:28:38 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:28:38 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:35:34 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.64.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "localhost"
2020/01/10 15:35:34 [warn] 6#6: *1 upstream server temporarily disabled while connecting to upstream, client: 192.168.64.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "localhost"
2020/01/10 15:35:34 [crit] 6#6: *1 connect() to [::1]:9000 failed (99: Address not available) while connecting to upstream, client: 192.168.64.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://[::1]:9000", host: "localhost"
2020/01/10 15:35:34 [warn] 6#6: *1 upstream server temporarily disabled while connecting to upstream, client: 192.168.64.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://[::1]:9000", host: "localhost"
2020/01/10 15:37:23 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "localhost:8001"
2020/01/10 15:37:23 [warn] 6#6: *1 upstream server temporarily disabled while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "localhost:8001"
2020/01/10 15:37:23 [crit] 6#6: *1 connect() to [::1]:9000 failed (99: Address not available) while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://[::1]:9000", host: "localhost:8001"
2020/01/10 15:37:23 [warn] 6#6: *1 upstream server temporarily disabled while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://[::1]:9000", host: "localhost:8001"
2020/01/10 15:37:57 [crit] 6#6: *4 connect() to [::1]:9000 failed (99: Address not available) while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://[::1]:9000", host: "localhost:8001"
2020/01/10 15:37:57 [warn] 6#6: *4 upstream server temporarily disabled while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://[::1]:9000", host: "localhost:8001"
2020/01/10 15:37:57 [error] 6#6: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "localhost:8001"
2020/01/10 15:37:57 [warn] 6#6: *4 upstream server temporarily disabled while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "localhost:8001"
2020/01/10 15:37:59 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost:8001"
2020/01/10 15:38:07 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "localhost"
2020/01/10 15:38:07 [warn] 6#6: *1 upstream server temporarily disabled while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "localhost"
2020/01/10 15:38:07 [crit] 6#6: *1 connect() to [::1]:9000 failed (99: Address not available) while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://[::1]:9000", host: "localhost"
2020/01/10 15:38:07 [warn] 6#6: *1 upstream server temporarily disabled while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://[::1]:9000", host: "localhost"
2020/01/10 15:38:09 [error] 6#6: *1 no live upstreams while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:39:32 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "localhost"
2020/01/10 15:39:32 [warn] 6#6: *1 upstream server temporarily disabled while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "localhost"
2020/01/10 15:39:32 [crit] 6#6: *1 connect() to [::1]:9000 failed (99: Address not available) while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://[::1]:9000", host: "localhost"
2020/01/10 15:39:32 [warn] 6#6: *1 upstream server temporarily disabled while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://[::1]:9000", host: "localhost"
2020/01/10 15:39:33 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:39:34 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/10 15:39:35 [error] 6#6: *4 no live upstreams while connecting to upstream, client: 192.168.80.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://localhost", host: "localhost"
2020/01/13 11:25:26 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:25:28 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:25:29 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:25:31 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:25:33 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:25:36 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:25:41 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:25:49 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:26:03 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:26:30 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:27:22 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:28:23 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:29:24 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:30:25 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:31:26 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:32:27 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:33:28 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:34:29 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:35:30 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:36:31 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:37:32 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:38:33 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:39:34 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:40:35 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:41:37 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:42:38 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:43:39 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:44:39 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:45:40 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:46:41 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:47:42 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:48:43 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:49:44 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:50:45 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:51:46 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:52:47 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:53:48 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:54:49 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:55:50 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:56:51 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:57:52 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:58:53 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 11:59:54 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 12:00:55 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
2020/01/13 12:01:56 [emerg] 1#1: host not found in upstream "cofrinhoapp" in /etc/nginx/conf.d/nginx.conf:9
